cuBERT_topic_modelling/tests/test_subwordtokenizer.py

```diff
@@ -0,0 +1,73 @@
+from transformers import AutoTokenizer, AutoModel
+import torch
+import numpy as np
+from cuBERTopic import gpu_BERTopic
+from sklearn.datasets import fetch_20newsgroups
+import cudf
+import pytest
+
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+
+# Sentences we want sentence embeddings for
+
+# TODO: sentence number 11927 is being encoded different in AutoTokenizer
+# and cuDF's SubwordTokenizer, look into that - this is causing high margin of error.
+
+sentences = fetch_20newsgroups(subset="all")["data"][:10000]
+
+
+def run_embedding_creation_transformers(sentences):
```

Comment:
we are failing `test_embedding_creation.py` so need to look into what's going on there.